internal/handlers/pg/pgdb/indexes.go

```diff
@@ -19,28 +19,46 @@ import (
 
 	"github.com/jackc/pgx/v4"
 
+	"github.com/FerretDB/FerretDB/internal/types"
 	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
 )
 
-// indexParams describes the parameters for creating an index.
+// indexParams contains parameters for creating an index.
 type indexParams struct {
-	schema   string // pg schema name
-	table    string // pg table name
-	isUnique bool   // whether the index is unique
+	db         string          // FerretDB database name
+	collection string          // FerretDB collection name
+	index      string          // FerretDB index name
+	key        *types.Document // Index specification (pairs of field names and sort orders)
```

Comment:
`key` is currently not used for index creation in postgres, which is okay I guess.